aytests/upgrade_sles11.install_xml

```diff
@@ -542,22 +542,22 @@ ls -l /mnt/$BASE
     <errors>
       <log config:type="boolean">true</log>
       <show config:type="boolean">true</show>
-      <timeout config:type="integer">0</timeout>
```

Comment:
I would keep this to 0. In order to see errors. May be not so harmful because the changes around sles11 should not happen
 very often.